students/jim/Lesson02/series.py

```diff
@@ -0,0 +1,26 @@
+def fibonacci_recursive(n):
+    '''
+    Recursively calculates the nth term in the Fibonacci sequence.
+    :param n: an int representing the nth sequential value in the series
+    :return: int
+    '''
+
+    if n > 2:
+        return fibonacci_recursive(n-2) + fibonacci_recursive(n-1)
+    return 1
+
+def fibonacci_iterative(n):
+    if n < 3:
+        return(1)
+    else:
+        fib_2 = 1
+        fib_1 = 1
+        for i in range(2, n):
+            value = fib_1 + fib_2
+            print(float(value) / float(fib_2))
+            fib_1 = fib_2
+            fib_2 = value
+        return(value)
+
+print(fibonacci_iterative(30))
+print(fibonacci_recursive(30))
```

Comment:
I got curious, and timed the code. Turns out for Fibonacci, recursive and iterative take nearly the same amount of time. I commented out the print statement when I tested, as that is a big time drain.